packages/easy-coding-standard/config/config.php

```diff
@@ -33,5 +33,7 @@
     $parameters->set(Option::PATHS, []);
     $parameters->set(Option::FILE_EXTENSIONS, ['php']);
 
+    $parameters->set(Option::REPORT_WARNINGS, []);
+
```

Comment:
This should be enabled by default, to include warnings as errors. So no configuration is needed.

Reply:
Removed but reverted, see d161513

Reply:
This is not needed. Sniff classes should be added in conatants, not here